yashsachdev.HabitTracker/DisplayClass.cs

```diff
@@ -0,0 +1,242 @@
+﻿namespace yashsachdev.HabitTracker;
+
+/// <summary>
+/// Display actions.
+/// </summary>
+public class DisplayClass
+{
+    public DisplayClass() { }
+    public bool LoggedIn { get; set; }
+    private int HabitId { get; set; }
+    private int UserId { get; set; }
+    public string Email { get; set; }
+    public void Register()
+    {
+        string name = UserInput.GetName();
+        this.Email = UserInput.GetEmail();
```

Comment:
Remove all "this" 
![image](https://user-images.githubusercontent.com/108711476/220470062-c7f63638-3e13-4868-97da-d8da3c3cda40.png)